libr/core/cmd_anal.c

```diff
@@ -7135,6 +7195,20 @@ static bool cmd_anal_refs(RCore *core, const char *input) {
 		r_list_free (list);
 		free (ptr);
 	} break;
+	case 'v': // "axv"
```

Comment:
I think we should be careful with not abusing too many commands... what's the advantage of having `axv` when there is already `afvR` and `afvW`? Also, is it possible to not use `r_core_cmd`?

Reply:
well, its about usability sometimes, and axv = analyze xrefs for variables, afv = display function variables. maybe axv could be renamed to afvx or afv=

Reply:
For me axv seems natural as `anal xref vars` but also i would think on `afvx` as in `anal function var xrefs`.